### DISCUSSION TITLE
striptracks script for radarr/sonarr

### DISCUSSION THREAD

**Discussion by di5cord20** [tteck/Proxmox#3814] [Help Needed]
Has anyone managed to install https://github.com/linuxserver/docker-mods/tree/radarr-striptracks on either Radarr or Sonarr?  Looking for some help to confirm if this is possible or only available for docker.  Thanks

**Comment by oOStroudyOo** [comment]
Do you run Radarr and Sonarr in Docker?
If not then this won't work for you.

I achieve the same thing via Tdarr.

**Reply by di5cord20** [reply]
I assumed as much but just wanted to confirm.
